src/hotspot/share/services/heapDumper.cpp

```diff
@@ -1955,7 +1955,9 @@ void DumpMerger::do_merge() {
       merge_file(path);
     }
     // Delete selected segmented heap file nevertheless
-    remove(path);
+    if (remove(path) != 0) {
+      log_info(heapdump)("Remove segment file (%d) failed (%d)", i, errno);
```

Comment:
Suggestion: "Removal of segment file ..."

Reply:
Fixed